packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -248,203 +254,190 @@ function BudgetCategoryPreview({ name, pending, style }) {
   );
 }
 
-class BudgetCategory extends PureComponent {
-  constructor(props) {
-    super(props);
+const BudgetCategory = memo(function BudgetCategory({
+  category,
+  editing,
+  index,
+  // gestures,
+  editMode,
+  blank,
+  style,
+  month,
+  onEdit,
+  onBudgetAction,
+  show3Cols,
+  showBudgetedCol,
+}) {
+  // let opacity = new Animated.Value(editMode || blank ? 0 : 1);
+  let opacity = editMode || blank ? 0 : 1;
 
-    let { editMode, blank } = props;
-    this.opacity = editMode || blank ? 0 : 1;
-  }
+  // useEffect(() => {
+  //   if (editing && ACTScrollViewManager) {
+  //     ACTScrollViewManager.setFocused(findNodeHandle(container.current));
+  //   }
+  // }, [editing]);
 
-  render() {
-    let {
-      category,
-      editing,
-      index,
-      style,
-      month,
-      onBudgetAction,
-      show3Cols,
-      showBudgetedCol,
-    } = this.props;
-    let budgeted = rolloverBudget.catBudgeted(category.id);
-    let balance = rolloverBudget.catBalance(category.id);
-    let spent = rolloverBudget.catSumAmount(category.id);
-
-    let content = !category.hidden && (
-      <ListItem
+  // useEffect(() => {
+  //   Animated.timing(this.opacity, {
+  //     toValue: this.props.editMode ? 0 : 1,
+  //     duration: 200,
+  //     easing: Easing.inOut(Easing.ease),
+  //   }).start();
+  // }, [editMode]);
+
+  let budgeted = rolloverBudget.catBudgeted(category.id);
+  let balance = rolloverBudget.catBalance(category.id);
+  let spent = rolloverBudget.catSumAmount(category.id);
+
+  let content = !category.hidden && (
+    <ListItem
+      style={{
+        backgroundColor: editing ? theme.altTableTextEditing : 'transparent',
+        borderBottomWidth: 0,
+        borderTopWidth: index > 0 ? 1 : 0,
+        ...style,
+      }}
+      data-testid="row"
+    >
+      <View style={{ flex: 1 }}>
+        <Text style={styles.smallText} data-testid="category-name">
+          {category.name}
+        </Text>
+      </View>
+      <View
         style={{
-          backgroundColor: editing ? theme.altTableTextEditing : 'transparent',
-          borderBottomWidth: 0,
-          borderTopWidth: index > 0 ? 1 : 0,
-          ...style,
+          alignItems: 'center',
+          flexDirection: 'row',
+          opacity: opacity,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
          opacity,
```